### DISCUSSION TITLE
Is Occlum 1.0 at this repo now?

### DISCUSSION THREAD

**Discussion by lidh15** [occlum/occlum#1071] [General]
It's believed that Occlum 1.0 had been released last weekend (10th Dec) at CNCC. But the release version is still 0.29. So I'd like to know is v1.0 really ready now?

**Comment by volcano0dr** [comment]
Thanks for your interest in occlum, we have pushed the `1.0.0-preview` branch and we expect to release 1.0.0-rc1 soon.
